### ISSUE TITLE
BUG: Categories Show as Undefined

### ISSUE THREAD

**Issue by milezzz** [TelescopeJS/Zeiss#3]
I thought maybe my ancient Debian 7.x VM was to blame but the issue persists after creating a new VM based on Debian 8.1.0 / Node v0.10.29 / Meteor 1.1.0.3.  I am using latest devel branch of Telescope, removed the hubble theme and meteor added Zeiss.  No errors on install.

To reproduce simply clone a fresh version of Zeiss and add a category.  It should show up as undefined like here:  http://i.imgur.com/HXqgAuc.jpg  

I wish I knew more how to troubleshoot this issue but I am only 48 hours into Meteor/Telescope :(

**Comment by SachaG** [comment]
If you're using Zeiss, I guess you must be running the `devel` branch of Telescope? I've been working on categories lately so it's possible I broke something… I'll look into it :)

**Comment by SachaG** [comment]
Oh also, do you have any errors in your client and/or server-side logs?

**Comment by milezzz** [comment]
@SachaG:  On the client I get:  "You called Router.path for a route named undefined but that route doesn't seem to exist. Are you sure you created it?"

Sadly I am not sure where to find the meteor server logs, could you enlighten me where I might find these?

**Comment by SachaG** [comment]
That's probably the issue. Did you have categories created before? And here's how to check logs: http://docs.telescopeapp.org/docs/troubleshooting-1

**Comment by milezzz** [comment]
> > http://docs.telescopeapp.org/docs/troubleshooting-1

Ahhh thanks, missed that in the docs.  Unfortunately when I do a:  `meteor logs telescope` it errors with:

`This site was deployed with an old version of Meteor that used site passwords instead of user accounts. Now we have a much better system, Meteor developer accounts.`

Hmm sounds like it thinks I am using meteor hosting which I definitely am not.  Damn server logs would be useful!

> > Did you have categories created before?

No, you should be able to replicate the error on a clean install of the devel branch of Telescope + Zeiss.

**Comment by SachaG** [comment]
Well if you typed `meteor logs` then yes, that's the command for checking logs when using Meteor's own hosting. The command for checking logs depends on your deployment method.

**Comment by milezzz** [comment]
This is all running in a Virtual Machine:  Debian 8.1.0 / Node v0.10.29 / Meteor 1.1.0.3 / MongoDB.  Installed everything from apt-get.

Even after Googling I'm still not sure where the log directives are for Meteor.  From the sounds of it I need to install a meteor package for proper server logging?  What do you guys use?

**Comment by SachaG** [comment]
Well, the logs just show up in the terminal where you run `meteor`. You don't really need a package to see them.

**Comment by milezzz** [comment]
Ahhh I see.  Was hoping there was more...

So on the server then I see no errors when I am adding a category at least.  

My guess is something in `devel` changed as you mentioned.  I'm pretty sure I was using it before and it was working as expected.  When I get some time I will try an older commit.

**Comment by SachaG** [comment]
OK, I found the bug :) Can you run `meteor update telescope:zeiss` and let me know if it's fixed for you?

**Comment by milezzz** [comment]
Hey that worked.  Categories are back, sweet!

As a side note...it seems the mouse cursor is acting a bit funky now.  Did you notice that?  Seems to flash on/off constantly when hovering links.  Tested in Chrome/Firefox in OS X.

Another thing I noticed was, I deleted a parent of a category that had a child and the child category becomes an orphan.  So it's there in the database but cannot be removed from the admin side.  Just an observation =)
